source/tyr/tyr/resources.py

```diff
@@ -1095,13 +1095,40 @@ def post(self, ac_instance_name):
         return marshal({'job': job}, one_job_fields), 200
 
 
+class DeleteDataset(flask_restful.Resource):
+
+    def delete(self, instance_name, type='poi'):
+
+        instance = models.Instance.get_by_name(instance_name)
+        if instance:
+            res = instance.delete_dataset(_type=type)
+            if res:
+                return_msg = 'All {} datasets deleted for instance {}'.format(type, instance_name)
+            else:
+                return_msg = 'No {} dataset to be deleted for instance {}'.format(type, instance_name)
+            return_status = 200
+        else:
+            return_msg = "No instance found for : {}".format(instance_name)
+            return_status = 404
+
+        return {'action': return_msg}, return_status
+
+
 class MigrateFromPoiToOsm(flask_restful.Resource):
-    def delete(self, instance_name):
 
+    def put(self, instance_name):
         instance = models.Instance.get_by_name(instance_name)
         if instance:
-            instance.delete_dataset()
```

Comment:
You still need to delete the dataset ot type poi

Reply:
It's error prone, if we keep two separate request the documentation have to be very clear.
remember that theses actions should be performed by members of the data team, that aren't developers

Reply:
I agree with @mbounabi  and propose to add "...an api tyr `PUT $HOST/v0/instances/$INSTANCE_NAME/actions/migrate_from_poi_to_osm` followed by DELETE $HOST/v0/instances/$INSTANCE_NAME/actions/delete_dataset/$TYPE  ..." in the file /ed/readme.md